.github/actions/javascript/createOrUpdateStagingDeploy/index.js

```diff
@@ -238,26 +238,39 @@ function getMergeLogsAsJSON(fromRef, toRef) {
 /**
  * Parse merged PRs, excluding those from irrelevant branches.
  *
- * @param {Array<String>} commitMessages
+ * @param {Array<Object<{commit: String, subject: String, authorName: String}>>} commits
  * @returns {Array<String>}
  */
-function getValidMergedPRs(commitMessages) {
-    return _.reduce(
-        commitMessages,
-        (mergedPRs, commitMessage) => {
-            if (!_.isString(commitMessage)) {
-                return mergedPRs;
-            }
+function getValidMergedPRs(commits) {
+    const mergedPRs = new Set();
+    _.each(commits, (commit) => {
+        const author = commit.authorName;
+        if (author === 'OSBotify') {
```

Comment:
Is there no constant for the OSBotify here?

Reply:
There is not. I can address that in a follow-up.

Reply:
I did this in a separate PR so we've got the CONST now.